simulationfactory/src/util/theme.html

```diff
@@ -0,0 +1,193 @@
+<div class="mdc-components">
```

Comment:
What is this file? Do we need it? How do we access it from the website?

Reply:
This was one of my attempts at importing a theme that I forgot to remove.  We definitely don't need it. I'll figure out how to remove it later today, or early tomorrow.